commands/user_e2e_test.go

```diff
@@ -136,3 +136,62 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestListUserCmd() {
```

Comment:
The tests fail on CI:
https://app.circleci.com/pipelines/github/mattermost/mmctl/1566/workflows/40e58ca7-452e-4a71-8133-a78352baca2e/jobs/5397/parallel-runs/0/steps/0-105

Could you please take a look what is going on?

Reply:
on it

Reply:
sorry, I forgot to add len on the asssertion, already adjusted
thanks @hanzei :+1: